tests/test_slogpy_file.py

```diff
@@ -0,0 +1,103 @@
+"""The purpose of the code is to give all methods a unit test to make sure that they are working properly"""
+import tempfile
+
+from slogpy.slog import Slog as slog  # noqa: N813
+
+
+def test_info_file(msg="This is a test"):
+    """Simple test for files output
+
+    Given:Using slogpy
+    When: Using info() with a String
+    Then: Contents of String shows up in a temp file AND nothing in stdout and stderr
+    """
+    # create temp file using tempfile
+    temp = tempfile.NamedTemporaryFile(delete=False)
```

Comment:
Suggestion: Clean up after yourself

Don't forget to delete any temp files created by your tests.  It's a good principle to 'leave no trace' for any automated tests meaning after they are finished executing, any artifacts they created are removed.

Reply:
It is also possible to use: https://docs.pytest.org/en/6.2.x/tmpdir.html
It is more inline with using pytest.  It does do some clean up, but doesn't appear to do a full clean up either.

Reply:
I changed the parameter delete = true.

Reply:
still not cleaning up